tutorials/setting-up-lamp/index.md

```diff
@@ -114,11 +114,9 @@ look up the address in the
 
     You should see the Apache test page.
 
-1. Create a test file in the default web server root at `/var/www/html/`. You can follow the instructions in the [php documentation](http://php.net/manual/en/tutorial.firstpage.php). Example number 2 is the simplest example.
+1. Create a test PHP file in the default web server root at `/var/www/html/`. You can follow the instructions in the [php documentation](http://php.net/manual/en/tutorial.firstpage.php). Or run the following shell command to create a quick `phpinfo.php` test file.
 
-    You can write the code to the file from the command line by using a statement like the following. Replace `[YOUR_PHP_CODE]` with the code you want to write out:
-
-        sudo sh -c 'echo "[YOUR_PHP_CODE]" > /var/www/html/phpinfo.php'
+        sudo sh -c 'echo "<?php echo phpinfo(); ?>" > /var/www/html/phpinfo.php'
```

Comment:
🤔 My suspicion is that `<?php echo phpinfo(); ?>` will be stripped out by the publishing platform's security, which disallows anything that looks like HTML.

Reply:
Yes, that's why it's written the way it is. 

Arun, you can revert this.